.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run process-readme
```

Comment:
What is the husky for?

Reply:
This provides the git hook to process the readme toc right before committing.